### PR TITLE
Variable declarations

### PR DESCRIPTION
**Pull request by Phantomcheg**

### CODE REVIEW

File: Exercises/variables-declaration/variables-declaration.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
+// 1. Declare a variable which will be encapsulated from global scope.
+// 3. Declare two variables which will be same-names but contain different values.
+let firstOne = 0;
+let secondOne = 'Some number';
+console.log(secondOne);
```
Comment (Mike-Savin) [inline Exercises/variables-declaration/variables-declaration.js]: 1. Are these vars really encapsulated from global (the top-most) scope? (It's not the same that to be not present in global object).
2. Which two variables have the same names but contain different values here?

**Comment by Mike-Savin** [comment]
Looks good in general)

File: Exercises/variables-declaration/variables-declaration.js (hunk 1 above)
Comment (Phantomcheg) [reply Exercises/variables-declaration/variables-declaration.js]: Committed some changes, hope now it's better)

Comment (Phantomcheg) [reply Exercises/variables-declaration/variables-declaration.js]: Thought that re-asigning the variable in the loop is the same

Comment (Phantomcheg) [reply Exercises/variables-declaration/variables-declaration.js]: Oh, I understand my mistake about encapsulation.
I encapsulated the value of variable in the loop, but not the variable itself

**Review by lizarusi — CHANGES_REQUESTED** [review]
All good, just first task with encapsulation require changes :) Feel free to ask if you have questions

**Review by Mike-Savin — APPROVED** [review]

**Pull request merged**